### PR TITLE
Changing Cart Page View

### PR DESCRIPTION
**Pull request by ula0156**
1) Made displayed products on cart page to be hyperlinks
2) Added "add copy", "remove copy"
3) Added method to cart controller "RemoveFromCart".

### CODE REVIEW

File: onlineShopWeb/Controllers/CartController.cs [hunk 1]
```diff
@@ -49,6 +49,21 @@ public ActionResult DisplayCart()
             return View(model);
         }
 
+        public RedirectToRouteResult RemoveFromCart(Guid id)
+        {
+            // when user wants to remove from cart:
+            // - find his cart by using identifier. Dictionary<Cart, identifier>
+            // - find corresponding product which has to be removed
+            // - call RemoveProduct method on cart, which also will take care of reservation.
+
```
Comment (simionliviumihai) [inline onlineShopWeb/Controllers/CartController.cs]: We should verify that the id is not null and return bad request if it is

File: onlineShopWeb/Models/CartViewModels.cs [hunk 2]
```diff
@@ -5,6 +5,7 @@ namespace onlineShopWeb.Models
 {
     public class CartViewModel
     {
+
```
Comment (simionliviumihai) [inline onlineShopWeb/Models/CartViewModels.cs]: Why empty line here?

**Review by simionliviumihai — APPROVED** [review]

File: onlineShopWeb/Controllers/CartController.cs (hunk 1 above)
Comment (ula0156) [reply onlineShopWeb/Controllers/CartController.cs]: Done

File: onlineShopWeb/Models/CartViewModels.cs (hunk 2 above)
Comment (ula0156) [reply onlineShopWeb/Models/CartViewModels.cs]: Removed

**Pull request merged**